src/Microsoft.AspNetCore.Razor.Runtime/project.json

```diff
@@ -56,9 +56,12 @@
     },
     "netstandard1.3": {
       "dependencies": {
+        "System.Collections.Concurrent": "4.0.12-*",
+        "System.Reflection": "4.1.0-*",
```

Comment:
Note: Highest version of `System.Reflection.Extensions` that we have available on cidev/volatiledev is 4.0.1 (which is what we're currently referencing). Therefore, doesn't look like there's currently any version to bump this up to.